contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
```

Comment:
@duglin aren't bindings namespaced as well? Generally, things that are namespaced can't refer to something in another namespace, so it's implicit that the namespace of the instance is the same as that of the binding.

But maybe, then, we should add a namespace in the metadata section?